server/model/organisation.go

```diff
@@ -9,23 +9,34 @@ import (
 // Organisation model definition
 type Organisation struct {
 	Base
-	Title            string  `gorm:"column:title" json:"title"`
-	Slug             string  `gorm:"column:slug" json:"slug"`
-	Description      string  `gorm:"column:description" json:"description"`
-	FeaturedMediumID *uint   `gorm:"column:featured_medium_id;default:NULL" json:"featured_medium_id"`
-	Medium           *Medium `gorm:"foreignKey:featured_medium_id" json:"medium"`
+	Title             string              `gorm:"column:title" json:"title"`
+	Slug              string              `gorm:"column:slug" json:"slug"`
+	Description       string              `gorm:"column:description" json:"description"`
+	FeaturedMediumID  *uint               `gorm:"column:featured_medium_id;default:NULL" json:"featured_medium_id"`
+	Medium            *Medium             `gorm:"foreignKey:featured_medium_id" json:"medium"`
+	Applications      *[]Application      `gorm:"foreignKey:organisation_id" json:"applications"`
+	OrganisationUsers *[]OrganisationUser `gorm:"foreignKey:organisation_id" json:"organisation_users"`
```

Comment:
change `*[]Application` to `[]Application`
change `*[]OrganisationUser` to `[]OrganisationUser`